packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -647,8 +697,19 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone> HoprCoreEthe
             .unwrap_or(Balance::zero(BalanceType::HOPR));
 
         let new_redeemed_balance = balance.add(&acked_ticket.ticket.amount);
-        self.db.set(key, &new_redeemed_balance).await?;
-        debug!("updated redeemed tickets value to {new_redeemed_balance}");
+        ops.put(key, &new_redeemed_balance);
+
+        let unrealized_balance = self.get_unrealized_balance(&acked_ticket.ticket.channel_id).await?;
+        let unrealized_balance_key = utils_db::db::Key::new_with_prefix(
+            &acked_ticket.ticket.channel_id,
+            TICKETS_CHANNEL_UNREALIZED_BALANCE_PREFIX,
+        )?;
+        ops.put(
+            unrealized_balance_key,
+            unrealized_balance.add(&acked_ticket.ticket.amount),
```

Comment:
When redeeming a ticket the unrealized balance should actually stay the same. This operation should reduce the balance in the channel, which is done once the changes are indexed by the indexer. So eventually the unrealized balance and actual balance will converge.